repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -1702,6 +1635,52 @@ export const upsertFareDayEnd = async (nocCode: string, fareDayEnd: string): Pro
         throw new Error(`Could not insert fare day end into the fareDayEnd table. ${error}`);
     }
 };
+export const getCapExpiry = async (nocCode: string): Promise<string | undefined> => {
```

Comment:
Please could you add some commented out examples in this functions of what the stringified version of the capValidity might be? It's hard to tell what we will get out of JSON.parse(string)